urchin_windows_config.toml.exe

```diff
@@ -0,0 +1,41 @@
+# Address to the MariaDB database
```

Comment:
This doesn't seem to have any windows-only changes from the `urchin_config.toml` file - do we need this? If not, we can also remove from PR

Reply:
Windows reads the config.toml only as a .exe in order to run Air.  =)

Reply:
I think you're mistaken here, this file is running from urchin so as long as `--config urchin_config.toml` is passed to urchin you should be fine. That's why I don't think we need this here.

For example, in your `.air_windows.toml` you just need

```
full_bin = ".\\tmp\\urchin.exe --config urchin_config.toml"
```